service/http.balancer/pom.xml

```diff
@@ -98,6 +98,24 @@
             <groupId>org.kaazing</groupId>
             <artifactId>k3po.junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Noted, thanks.  This is a common mistake I try and correct when I see it.  Also the generally in " there is generally no code in them that you need while compiling the tests" is not always true.